df.history.xml

```diff
@@ -200,8 +200,43 @@
             <stl-vector name="wanted">
                 <pointer>
                     <int32_t name='entity_id' ref-target='historical_entity'/>
-                    <stl-vector type-name='int32_t'/>
-                    <stl-vector type-name='int32_t'/>
+                    <stl-vector name='types'>
+                        <enum base-type='int32_t'>
+                            0
+                            <enum-item name='Hero'/>
```

Comment:
A more complete list, extracted from what looks to be conversation-related code:
0 - hero
1 - close friend
2 - really irritating
3 - my partner
4 - brawler
5 - villain
6 - good for a deal now and then
7 - a pleasure to speak with
8 - killer
9 - murderer
10 - comrade
11 - worthy of respect
12 - not fit to rule
13 - enemy fighter
14 - on my side
15 - thug
16 - brigand
17 - loyal soldier
18 - monster
19 - storyteller
20 - poet
21 - bard
22 - dancer
23 - quarreler
24 - flatterer
25 - hunter
26 - protector of the defenseless